lib/BloqadeSchema/src/types.jl

```diff
@@ -114,6 +117,7 @@ end
     n_shots::Int = 1
 
     warn::Bool = false
+    discretize::Bool = true
```

Comment:
This could be confusing since the waveforms are also discretized.

Reply:
Well the one thing that would be good to use this name is to be consistent with the AWS SDK which uses the same optional argument. We might want to rename the function in BloqadeWaveforms.